bitnami/keycloak/templates/_helpers.tpl

```diff
@@ -17,6 +17,17 @@ If release name contains chart name it will be used as a full name.
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return the proper Keycloak namespace
+*/}}
+{{- define "keycloak.namespace" -}}
```

Comment:
There is a helper called "common.names.namespace". Could you use this instead of creating this new helper?

Reply:
Fixed! Thank you for the review and pointing me in the right direction for the changes!